ignite/distributed/comp_models/native.py

```diff
@@ -371,8 +371,14 @@ def spawn(  # type: ignore[override]
 
             start_processes = mp.spawn
             # start_method and start_processes in pytorch >= 1.5
+            default_start_method = "spawn"
             if LooseVersion(torch.__version__) >= LooseVersion("1.5.0"):
-                spawn_kwargs["start_method"] = kwargs.get("start_method", "spawn")
+
+                if "__IPYTHON__" in globals():
+                    # use fork in jupyter
+                    default_start_method = "fork"
+                spawn_kwargs["start_method"] = kwargs.get("start_method", default_start_method)
+                kwargs["start_method"] = "fork"
```

Comment:
Why does `kwargs["start_method"]` is forced to `fork` ? It could be not consistent with `spawn_kwargs["start_method"]`. Sorry, I think I missed something.

Reply:
Yes, I agree, this is wrong.